### PR TITLE
Small README changes

### PR DESCRIPTION
**Pull request by rwegener2**
This PR makes two small suggestions to the README:

1. Fixes a typo
2. Adds description of how to run tests

For the tests description of course please re-write, move to another section, or change as you see fit (you should have edit access to this branch). Just wanted to put in a placeholder.

https://github.com/openjournals/joss-reviews/issues/7406

### CODE REVIEW

File: README.md
Comment (rwegener2) [inline README.md:1]: This is really interesting, but I worry it is a bit dense for someone with less programming background. I would consider adding some higher level text here before the denser paragraph, for example maybe
"NdbcApi provides a singular access point for NDBC data. All the methods for finding data, metadata, and locating stations are available using the api object."

**Comment by rwegener2** [comment]
One more comment -- I found it took a moment to understanding "modes". I will suggest a more thorough explanation of modes in the example notebook, but 1-2 short sentences explaining that data variables are grouped into modes here in the README may also be helpful.

File: README.md
Comment (CDJellen) [reply README.md:1]: Thank you for this suggestion; I agree and have included this change in the PR you opened.

While I enjoyed building these features when initially designing the API, they are all but irrelevant to end users.

Additionally, I've split off the test instructions into a new `CONTRIBUTING.md` guide.

**Comment by CDJellen** [comment]
> One more comment -- I found it took a moment to understanding "modes". I will suggest a more thorough explanation of modes in the example notebook, but 1-2 short sentences explaining that data variables are grouped into modes here in the README may also be helpful.

Thank you for pointing this out; I've made some updates to the README to contextualize each of the mode identifiers.

**Review by CDJellen — APPROVED** [review]

**Pull request merged**